docs/data/material/discover-more/backers/backers.md

```diff
@@ -2,11 +2,9 @@
 
 <p class="description">Support the development of new open-source projects from MUI through crowdfunding.</p>
 
-The core of MUI is our crowd-funded open-source projects, licensed under the permissive MIT license. Sponsorship increases the rate of bug fixes, documentation improvements, and feature development.
+MUI Core is our crowd-funded open-source projects, licensed under the permissive MIT license. Sponsorship increases the rate of bug fixes, documentation improvements, and feature development.
```

Comment:
```suggestion
MUI Core contains our crowd-funded open-source projects, offered under the permissive MIT license. Sponsorship increases the rate of bug fixes, documentation improvements, and feature development.
```

Reply:
I believe we have 3 options when it comes to the Open Collective. I went with a different approach for this page description, using option 2 below:

1. Creating one open-collective per open-source project: https://opencollective.com/mui is about Material UI, we would have a couple more to create. Considering the traction we have with the community, and the impact these donations tend to have (low), we have time until we need ones. So effectively, only rename https://opencollective.com/mui to https://opencollective.com/material-ui for now.
2. Considering that MUI is the organization, the umbrella under which we are pushing for the work. It doesn't feel so great, as a developer would likely want to sponsor one project and not another, like Vite vs. Vitest. However, it stays broad, still allowing us to cover some of Base UI, Joy UI, and MUI X.
3. Considering that MUI is one open-source project that has a large breadth. In which case, https://opencollective.com/mui wouldn't need any changes.

Even if we kill https://mui.com/joy-ui/, and we split Joy to be under Material UI (as a theme, like Material Design v3, or Material Design v2, while we work on an infra to create more theme), as well as under Base UI (as a theme), it feels like option 2 still makes the most sense right now. 

The problem with option 3 is how much it can be perceived by developers as a. lacking focus and locked-in. For example, when you look at Slack vs. Microsoft Team, you are more likely to use Slack as a standalone product, but overall Microsoft Team wins because it comes for free in a bundle you already purchased. So if we were to apply this straight to us, option 3 would make the most sense. However, our space is very different: 1. products are free 2. product value compounds with its community size. We are only left with the value of bundling. The main problem is how developers adopt the solutions that have the best execution. Headless UI failure is an example of this, Radix had x6 more execution, it was too far for Headless UI to thrive with its Tailwind CSS users.

One thing, maybe https://opencollective.com/mui -> https://opencollective.com/mui-org would make that intent to be clearer. In the same line of thought, going to GitHub https://github.com/mui/ -> https://github.com/mui-org/.

I think it's not to be confused with a change on npm @mui/material -> @material-ui/core, because there the npm org name is expected to be the open-source project name (I'm not aware of any project where it's not the case, we are the exception). On GitHub the common expectation is for it to be anything type, so we are good there.

cc @mbrookes @danilo-leal @mnajdova 

---

We are moving toward "MUI Core" to a notion of "Core of MUI", so I used more direct references to these open-source projects.